InterfaceParser/model/interface_item_base.py

```diff
@@ -0,0 +1,42 @@
+"""Base class for interface item.
+
+"""
+
+from abc import ABC
+
+
+class InterfaceItemBase(ABC):
+    """Base class for interface item.
+
+    :param name: item name
+    :param description: list of string description elements
+    :param design_description: list of string design description elements
+    :param issues: list of issues
+    :param todos: list of string todo elements
+    :param platform: optional platform (string or None)
+    :param default_value: optional default value of this parameter
+    :param scope: optional scope: internal, partner or none (none by defaul, means public)
+    :param since:
+    :param until:
+    :param deprecated:
+    :param removed:
+    :param history:
+
+    """
+
+    def __init__(self, name, description=None, design_description=None,
+                 issues=None, todos=None, platform=None, default_value=None, scope=None,
+                 since=None, until=None, deprecated=None, removed=None, history=None):
+        self.name = name
+        self.description = description if description is not None else []
+        self.design_description = design_description if design_description is not None else []
```

Comment:
Is design_description used anywhere in the project?

Reply:
this attribute used in sdl_core generators:
```
sdl_core/tools/InterfaceGenerator/generators/SmartFactoryBase.py
sdl_core/tools/InterfaceGenerator/generators/PolicyTypes.py
```
Note sdl_core going to be based on mentioned parsed/model from rpc_spec.